pkg/apis/v1alpha1/register.go

```diff
@@ -38,26 +38,45 @@ var (
 	RestrictedLabelDomains = []string{
 		LabelDomain,
 	}
-	AMIFamilyBottlerocket = "Bottlerocket"
-	AMIFamilyAL2          = "AL2"
-	AMIFamilyUbuntu       = "Ubuntu"
-	AMIFamilyCustom       = "Custom"
-	SupportedAMIFamilies  = []string{
+	AMIFamilyBottlerocket  = "Bottlerocket"
+	AMIFamilyAL2           = "AL2"
+	AMIFamilyUbuntu        = "Ubuntu"
+	AMIFamilyWindowsServer = "WindowsServer"
```

Comment:
```suggestion
	AMIFamilyWindows = "Windows"
```

I think we probably just want to keep the naming as short as possible

Reply:
Sure.